content/docs/reference/public_private_keys/index.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: single
+
+title: 'Public and Private Keys' # The title (ON THE PAGE)
+lead: | # The lead below the title (ON THE PAGE)
+  <em>Providing access to information</em>
```

Comment:
Try to keep frontmatter consistent, one lead is italicized, the other isn't. I would say don't italicize.